extensions/twitch/src/following.tsx

```diff
@@ -67,14 +61,11 @@ export default function main() {
           <ActionPanel>
             <OpenAction title="Open Channel" target={`https://twitch.tv/${item.user_name}`} />
             <OpenAction title="Open Stream in Streamlink" target="streamlink" onOpen={(target) => {
-              if (item.type != "live") { confirmAlert({ title: "Info", message: "This streamer is offline" }); return; }
+              if (item.type != "live") { showToast(ToastStyle.Failure, "This streamer is offline!"); return; }
 
               exec(`${streamlinkLocation} https://twitch.tv/${item.user_name} ${quality} ${playerLocation ? `--player ${playerLocation}` : ""}`, (err, stdout, stderr) => {
```

Comment:
passing streamlinklocation twitch.tv/streamername quality and --playerlocation doesn't seem to work reliably at least with IINA and Quicktime player.


```ts
  async function startLowLatencyStream(streamURL) {
    const command = `${streamlinkLocation} twitch.tv/${streamURL} ${quality} --twitch-low-latency`;
    console.log(command);

    exec(command, (error, stdout, stderr) => {
      if (error) {
        showToast(ToastStyle.Failure, "Failed to start stream");
      }
      if (stderr) {
        showToast(ToastStyle.Failure, stderr);
      }
    });
  }

  function startStream(m3u8_url) {
    const command = `open -a "Quicktime Player" "${m3u8_url}"`;
    console.log(command);

    exec(command, (error, _, stderr) => {
      if (error) {
        showToast(ToastStyle.Failure, "Failed to start stream");
      }
      if (stderr) {
        showToast(ToastStyle.Failure, stderr);
      }
    });
  }

  function streamUsingM3U8(user_name) {
    const command = `${streamlinkLocation} twitch.tv/${user_name} ${quality} --stream-url`;
    console.log(command);

    exec(command, (error, stdout, stderr) => {
      console.log(stdout);
      if (error) {
        showToast(ToastStyle.Failure, "Failed to start stream");
      }
      if (stderr) {
        showToast(ToastStyle.Failure, stderr);
      }

      if (stdout.trim().endsWith(".m3u8")) {
        startStream(stdout);
      } else {
        showToast(ToastStyle.Failure, "Stream is not available");
      }
    });
  }
```